src/sideshift.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import type {StandardTx, SwapFuncParams} from './checkSwapService'
+
+const {checkSwapService} = require('./checkSwapService.js')
+const js = require('jsonfile')
+const fetch = require('node-fetch')
+
+const confFileName = './config.json'
+const config = js.readFileSync(confFileName)
+
+const SIDESHIFT_CACHE = './cache/XAIRAW.json'
+const PAGE_LIMIT = 500
+
+type SideShiftTransaction = {
```

Comment:
`SideShiftTransaction` or `SideshiftTransaction`?

`sideShift` or `sideshift`?

@abrkn: we need to get these casing rules right.